doc/classes/ReflectionProbe.xml

```diff
@@ -28,17 +34,8 @@
 		<member name="intensity" type="float" setter="set_intensity" getter="get_intensity" default="1.0">
 			Defines the reflection intensity. Intensity modulates the strength of the reflection.
 		</member>
-		<member name="interior_ambient_color" type="Color" setter="set_interior_ambient" getter="get_interior_ambient" default="Color( 0, 0, 0, 1 )">
-			Sets the ambient light color to be used when this probe is set to [member interior_enable].
-		</member>
-		<member name="interior_ambient_contrib" type="float" setter="set_interior_ambient_probe_contribution" getter="get_interior_ambient_probe_contribution" default="0.0">
-			Sets the contribution value for how much the reflection affects the ambient light for this reflection probe when set to [member interior_enable]. Useful so that ambient light matches the color of the room.
-		</member>
-		<member name="interior_ambient_energy" type="float" setter="set_interior_ambient_energy" getter="get_interior_ambient_energy" default="1.0">
-			Sets the energy multiplier for this reflection probe's ambient light contribution when set to [member interior_enable].
-		</member>
-		<member name="interior_enable" type="bool" setter="set_as_interior" getter="is_set_as_interior" default="false">
-			If [code]true[/code], reflections will ignore sky contribution. Ambient lighting is then controlled by the [code]interior_ambient_*[/code] properties.
+		<member name="interior" type="bool" setter="set_as_interior" getter="is_set_as_interior" default="false">
+			If [code]true[/code], reflections will ignore sky contribution.
```

Comment:
I don't know if the new `ambient_*` properties serve the same purpose as the removed `internal_ambient_*`, but if they do, there are relevant docstrings to port here.